.dockerignore

```diff
@@ -0,0 +1,5 @@
+**/.git
+**/.gitmodules
+**/.travis.yml
+**/Dockerfile
+**/*.md
```

Comment:
you can probably exclude other stuff here:
- the travis folder
- I added a docker file called Dockerfile.grpc
- ...
but that's up to you